docs/usage.rst

```diff
@@ -47,16 +47,36 @@ We select the :class:`audbackend.backend.FileSystem` backend.
     audbackend.backend.FileSystem.create("./host", "repo")
 
 Once we have an existing repository,
-we can access it by instantiating the backend class.
-In addition,
-we use :class:`audbackend.interface.Unversioned`
-as an interface to the backend.
+we can access it by instantiating the backend class
+and open a connection by calling
+:meth:`audbackend.backend.open`.
+Later on,
+we will use
+:meth:`audbackend.backend.close`
+to close the connection.
+Instead of explicitly calling
+:meth:`audbackend.backend.open`
+and
+:meth:`audbackend.backend.close`
+it is good practise to use a with statement
+(not applicable in a notebook).
+
+.. jupyter-execute::
+
+    backend = audbackend.backend.FileSystem("./host", "repo")
+    backend.open()
+
+After establishing a connection
+we could directly execute read and write operations
+on the backend object.
+However,
+here we use :class:`audbackend.interface.Unversioned`
```

Comment:
```suggestion
However,
we recommend to always use :mod:`interfaces <audbackend.interface>`
to communicate with a backend.
Here, we use :class:`audbackend.interface.Unversioned`
```

Reply:
Should we mention that it behaves exactly like calling the methods directly on `backend`?

Reply:
No, I think this doesn't matter as a user should anyway not directly interact with the backend object.

Reply:
![Screenshot from 2024-04-02 15-01-14](https://github.com/audeering/audbackend/assets/10383417/d225aaac-0363-4842-9a77-4c9faf522d0f)